gw_eccentricity/eccDefinition.py

```diff
@@ -1539,17 +1543,17 @@ def get_orbit_averaged_omega22_at_pericenters(self):
         return (np.diff(self.phase22[self.pericenters_location]) /
                 np.diff(self.t[self.pericenters_location]))
 
-    def get_orbit_averaged_omega22_at_apocenters(self):
+    def get_orbit_averaged_omega22_between_apocenters(self):
         """Get orbital average of omega22 between two consecutive apocenters.
 
         The procedure to get the orbital average of omega22 at apocenters is
```

Comment:
It still says "at apocenters" here...please check carefully for other instances.

Reply:
Done